docs/adr/0029-sunset-osi-interactive.md

```diff
@@ -0,0 +1,42 @@
+# 29. Sunset OpenSAFELY Interactive
+
+Date: 2025-03
+
+## Status
+
+Accepted.
+
+## Context
+
+OpenSAFELY Interactive (OSI) was a Job Server app allowing users to request
+simple analyses.  We are removing OSI.
+
+Refer to the Team Manual for more information:
+
+https://github.com/ebmdatalab/team-manual/blob/main/docs/products/opensafely-interactive.md
+https://bennett.wiki/products/opensafely-interactive/
```

Comment:
I think it should either be only the link to the wiki, or a permalink of the current commit of that file, incase we move it or change to a CMS in the future.